### PR TITLE
quacs-rs: fix cargo-clippy lints

### PR DESCRIPTION
**Pull request by io12**

### CODE REVIEW

**Review by elihschiff — APPROVED** [review]
I don't know anything about rust but I trust you

**Comment by benjaminrsherman** [comment]
Would it be possible to add this to the lint github action?

**Comment by io12** [comment]
> Would it be possible to add this to the lint github action?

Is there an easy way to make `yarn lint` run clippy? The current thing is
```json
"lint": "vue-cli-service lint",
```
I'm not sure if there's a way to hook into `vue-cli-service` or if there's a better way.

**Comment by benjaminrsherman** [comment]
The commands in `package.json` are just shell commands, so you could run clippy as well with `"lint": "vue-cli-service lint && ..."`.

**Comment by io12** [comment]
Ok, hopefully the CI has clippy installed

**Comment by io12** [comment]
Oh it failed because it appends `--no-fix --max-warnings 0`. I guess I can switch the order from `vue-cli-service lint && cargo clippy --manifest-path src/quacs-rs/Cargo.toml -- -D warnings` to `cargo clippy --manifest-path src/quacs-rs/Cargo.toml -- -D warnings && vue-cli-service lint`?

File: site/scripts/lint.sh [hunk 1]
```diff
@@ -0,0 +1,18 @@
+#!/bin/bash
+
+set -eu
+
+echo "Running vue linter"
+vue-cli-service lint "$@"
```
Comment (benjaminrsherman) [inline site/scripts/lint.sh:6]: This should exit the script if it fails

**Review by benjaminrsherman — CHANGES_REQUESTED** [review]

File: site/scripts/lint.sh (hunk 1 above)
Comment (io12) [reply site/scripts/lint.sh:6]: I think `set -e` does this.

Comment (benjaminrsherman) [reply site/scripts/lint.sh:6]: Oh right, my bad

**Review by benjaminrsherman — APPROVED** [review]

**Pull request merged**